test/unit/warp/src/Activemask.cpp

```diff
@@ -53,6 +54,15 @@ struct ActivemaskMultipleThreadWarpTestKernel
     }
 };
 
+namespace alpaka::trait
+{
+    template<std::uint32_t TWarpSize, typename TAcc>
+    struct WarpSize<ActivemaskMultipleThreadWarpTestKernel<TWarpSize>, TAcc>
+    {
+        static constexpr std::uint32_t warp_size = TWarpSize;
+    };
+} // namespace alpaka::trait
+
```

Comment:
```suggestion
template<std::uint32_t TWarpSize, typename TAcc>
struct alpaka::trait::WarpSize<ActivemaskMultipleThreadWarpTestKernel<TWarpSize>, TAcc>
{
    static constexpr std::uint32_t warp_size = TWarpSize;
};
```
should work and saves a few LOCs. occurs more times in this PR.